src/components/metrics/card/index.js

```diff
@@ -178,10 +190,26 @@ class MetricCard extends Component {
     return errors ? errors.filter(e => !!e) : []
   }
 
+  showSimulation() {
+    const stats = _.get(this.props, 'metric.simulation.stats')
+    if (stats && _.isFinite(stats.mean) && _.isFinite(stats.stdev) && _.isFinite(stats.length)) {
+      return (stats.stdev === 0 || (stats.length > 5))
+    } else {
+      return false
+    }
+  }
+
+  _shouldShowSensitivitySection() {
+    const isAnalysis = (this.props.canvasState.metricCardView === 'analysis')
+    return isAnalysis && this.showSimulation() && this.props.selectedMetric
```

Comment:
Does this return a boolean or an object? Maybe make `this.props.selectedMetric` `!!this.props.selectedMetric`?